idl/chromadb/proto/chroma.proto

```diff
@@ -286,10 +287,16 @@ service VectorReader {
     rpc QueryVectors(QueryVectorsRequest) returns (QueryVectorsResponse) {}
 }
 
+message RequestMetadata {
+    int32 collection_version = 1;
```

Comment:
How do we feel about further propagating the pattern that all segments are concurrently compacted, I am fine with it but just want to ask?

Reply:
The collection version change is to identify a successful compaction. We can still use the collection version even if we compact a specific segment to identify new successful compaction.